packages/core/src/actors/promise.ts

```diff
@@ -91,8 +101,32 @@ export function fromPromise<T, TInput>(
         return;
       }
 
+      function step(name: string, promiseCreator: () => Promise<any>) {
+        if (state.steps?.[name]) {
+          return state.steps[name][1];
+        }
+
```

Comment:
Could the step be visualized as "pending" when something like a `$$xstate.step.pending` event would be sent?